### ISSUE TITLE
VP9 Alpha channel isn't decoded

### ISSUE THREAD

**Issue by Zipdox** [mpv-player/mpv#10304]
### Important Information
- mpv 0.34.1
- Debian GNU/Linux bookworm/sid x86_64
- Installed with apt

### Reproduction steps
Make sure your system supports transparency rendering. On Linux you might need a build of mesa with [these patches](https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/9989).

Refer to the example files. Play these two files with the following commands. The gif will play transparently but the WebM will not.
```sh
mpv --alpha=yes travolta.webm
```
```sh
mpv --alpha=yes Nikki.gif
```

### Expected behavior
The WebM plays with transparency.

### Actual behavior
The WebM plays with a black background.

### Log file
[mpv_transparency.log](https://github.com/mpv-player/mpv/files/8934412/mpv_transparency.log)

### Sample files

- travolta.webm: https://0x0.st/ouvI.webm
- Nikki.gif: https://0x0.st/ouvl.gif

**Comment by Zipdox** [comment]
It appears that forcing the decoder using `--vd=libvpx-vp9` works as a workaround, though it would be nice if this is not necessary.

**Comment by Zipdox** [comment]
Yeah I was aware that FFmpeg does this too, and you need to specify the decoder to decode alpha. Maybe MPV could detect alpha channel and force the decoder of necessary.

**Comment by TeeMoYan** [comment]
Can you play alpha webm?

**Comment by Zipdox** [comment]
> Can you play alpha webm?

Only if I install the mesa patches and force the encoder like I said.
